website/static/img/package.svg

```diff
@@ -0,0 +1,4 @@
+<!-- Copyright 2000-2021 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSES file. -->
```

Comment:
If its just this image under jetbrains apache license, maybe we should mention that in the license file (-name)? I'm not a lawyer what we can/have to/mustn't do

Reply:
I'm not a lawyer, too  
I guess it's fine, but if you want me to add it, I can do that